### ISSUE TITLE
Usability/Accessibility: scrollbar handle min-hight

### ISSUE THREAD

**Issue by bnjmnrsh** [bnjmnrsh-projs/signal-v-noise#17]
The scroll bar handle can be come very small on some tablets. A fixed, or minimum height should be set. 

![Screenshot 2022-11-22 at 12 57 55](https://user-images.githubusercontent.com/65979639/203319938-9b18c1d9-622e-414b-89c3-16824cc260d9.png)
